<?xml version="1.0"?>

<launch>
    <!--<include file="$(find robot_description)/launch/sim_w1.launch"></include>-->


    <arg name="des_x" default= "0"/>
    <arg name="des_y" default= "0"/>
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="mobile_robot_navigation_project" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="mobile_robot_navigation_project" type="go_to_point_service.py" name="go_to_point" />
    <node pkg="mobile_robot_navigation_project" type="bug0.py" name="bug" output="screen"/>
</launch>
